tests/system/python/packages/test_available_and_install_api.py

```diff
@@ -126,10 +135,17 @@ def test_install_service_package(self, foglamp_url):
         r = r.read().decode()
         jdoc = json.loads(r)
         assert len(jdoc), "No data found"
+        if 3 == len(jdoc['services']):
```

Comment:
CSV report status on the basis of HTTP_STATUS_CODE; if 200 then Passed otherwise Failed. And right place of this check is before HTTP status code assertion